### PR TITLE
Mask nebula passwords

### PR DESCRIPTION
**Pull request by yassingh**
Currently, some passwords are also getting logged by hubble osqueries.
This PR aims to mask such passwords. 

See the corresponding PR in hubble code for more details:
https://github.com/hubblestack/hubble/pull/429

DO NOT MERGE. 
( still need to test a few scenarios )
But reviews are welcome.

### CODE REVIEW

File: hubblestack_nebula_v2/top.mask [hunk 1]
```diff
@@ -0,0 +1,3 @@
+nebula:
+  '*':
```
Comment (basepi) [inline hubblestack_nebula_v2/top.mask]: I recently changed the nebula get_top_data to expect the matches as a list of dictionaries. This way ordering is guaranteed. I think you need to change this line as a result:

```
  - '*':
```

**Comment by yassingh** [comment]
Changes are complete. The code is good to be merged ( from my side atleast ).
Hence removed the 'do not merge' label.

**Comment by basepi** [comment]
Waiting on 
https://github.com/hubblestack/hubble/pull/429

File: hubblestack_nebula_v2/top.mask (hunk 1 above)
Comment (yassingh) [reply hubblestack_nebula_v2/top.mask]: Done

**Pull request merged**